src/Tokenizer/Tokens.php

```diff
@@ -1083,6 +1083,13 @@ public function isMonolithicPhp()
             ) {
                 return false;
             }
+
+            if ($this[$index]->isGivenKind(T_CLOSE_TAG)) {
```

Comment:
I think that this should be separated PR.
One PR with descriptions, one with logic change